doc/source/user_guide/scale.rst

```diff
@@ -358,6 +358,7 @@ results will fit in memory, so we can safely call ``compute`` without running
 out of memory. At that point it's just a regular pandas object.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
This is due to an empty Series being instantiated in `dask/dataframe/core.py:2058`

Reply:
this is ok, can you create an issue on dask dev about changing these defaults (and give this example), reference this PR.

Reply:
xref: https://github.com/dask/dask/issues/5630

Reply:
can you also create an issue on the pandas side so we know what we need to update once dask is at a certain version (you can just point to this line here)